doc/tutorials/menu/menu_01.py

```diff
@@ -0,0 +1,38 @@
+"""
+Menu.
+"""
+import arcade
+
+# Screen title and size
+SCREEN_WIDTH = 800
+SCREEN_HEIGHT = 600
+SCREEN_TITLE = "Making a Menu"
+
+
+class MainView(arcade.View):
+    """ Main application class."""
+
+    def __init__(self):
+        super().__init__()
+
+    def on_show_view(self):
+        """ This is run once when we switch to this view """
+        arcade.set_background_color(arcade.color.DARK_BLUE_GRAY)
+
+    def on_draw(self):
+        """ Render the screen. """
+        # Clear the screen
+        self.clear()
+
+
+def main():
+    """ Main function """
+
```

Comment:
For in-text examples, probably drop some of the extra comments like the ones above.

Reply:
Updated in https://github.com/pythonarcade/arcade/pull/1608/commits/438501342233c8ff34f90907e395e29fc90d2d9e